<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogCoordinateDirectsolution</class>
 <widget class="QDialog" name="DialogCoordinateDirectsolution">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>315</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>坐标正算</string>
  </property>
  <widget class="QGroupBox" name="groupBox_18">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>531</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>单点转换</string>
   </property>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>201</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>大地坐标</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>181</width>
       <height>80</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label_Trans_Soruce_X">
          <property name="text">
           <string>纬度：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_Trans_Source_X">
          <property name="inputMethodHints">
           <set>Qt::ImhDigitsOnly</set>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="label_Trans_Soruce_Y">
          <property name="text">
           <string>经度：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_Trans_Source_Y">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QLabel" name="label_Trans_Soruce_Z">
          <property name="text">
           <string>大地高：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_Trans_Source_Z">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_6">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>20</y>
      <width>191</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>高斯坐标</string>
    </property>
    <widget class="QWidget" name="layoutWidget_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>171</width>
       <height>80</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_11">
        <item>
         <widget class="QLabel" name="label_Trans_Target_X">
          <property name="text">
           <string>北：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_Trans_Target_X">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_12">
        <item>
         <widget class="QLabel" name="label_Trans_Target_Y">
          <property name="text">
           <string>东：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_Trans_Target_Y">
          <property name="text">
           <string/>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_13">
        <item>
         <widget class="QLabel" name="label_Trans_Target_Z">
          <property name="text">
           <string>高程：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_Trans_Target_Z">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget_8">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>50</y>
      <width>83</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_Trans_Zone">
       <property name="text">
        <string>带号：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_Trans_Zone">
       <property name="inputMask">
        <string notr="true"/>
       </property>
       <property name="text">
        <string notr="true">50N</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton_Trans_TranCoor">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>80</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>转换坐标</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>621</width>
     <height>20</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>说明：逗号分隔符文件（CSV），格式为“点号,纬度,经度,椭球高”，椭球高可省略！</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>531</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>投影设置</string>
   </property>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>20</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_22">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <widget class="QLabel" name="label_27">
       <property name="text">
        <string>东偏移：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_Trans_FalseEast">
       <property name="text">
        <string>500000</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_3">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>20</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_23">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <widget class="QLabel" name="label_28">
       <property name="text">
        <string>北偏移：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_Trans_FalseNorth">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Trans_MiddleLon">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>20</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>117</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_Trans_Project">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>国家2000</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>北京54</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>西安80</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>WGS84</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>UTM</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>使用七参数</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>坐标系统：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Trans_MiddleLon">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>20</y>
      <width>72</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>中央子午线：</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>531</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>批量转换</string>
   </property>
   <widget class="QWidget" name="layoutWidget_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>511</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_24">
     <item>
      <widget class="QPushButton" name="pushButton_Tans_SelectFile">
       <property name="text">
        <string>选择文件</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_Trans_FilePath">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Tans_TansFile">
       <property name="text">
        <string>转换</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
